doc/manual/problem-format.rst

```diff
@@ -12,6 +12,9 @@ On top, DOMjudge defines a few extensions:
    distributed to participants. The file extension determines any of
    three supported formats. If multiple files matching this pattern are
    available, any one of those will be used.
+ * Annotation with ``EXPECTED_RESULTS`` for jury solutions. The ICPC
+   problem package specification always has precedence. Use this to annotate
+   possible outcomes a submission can have to use together with the Judging verifier. 
```

Comment:
It would be good to clarify how these tags should be used exactly, and maybe also refer to a sample submission.
BTW, if you're importing submissions from a problem package, then they have to be located inside a `submissions/*` directory, so will always be overridden, I guess.

Reply:
Agreed that a clarification on the usage would be nice. Perhaps a link to https://github.com/Kattis/problem-package-format/pull/4 suffices, but I would at least write the tag in the exact required format (i.e., two `@`s and a space after the `:`).

Indeed, the `@EXPECTED_RESULTS@: ` tag is overridden when the submission is in one of the standard submission directories (which are `submissions/{accepted,wrong_answer,time_limit_exceeeded,run_time_error}/`. Perhaps we can explicitly state that this tag therefore only works in non-standard submission directories (like `mixed/` or `rejected/`).

Reply:
> maybe also refer to a sample submission

I also agree with this one, but the current example submissions in https://github.com/DOMjudge/domjudge/tree/main/example_problems do not exactly match the spec, so linking to them in their current state may be confusing. For many of these submissions, the `@EXPECTED_RESULTS@: ` tag is ignored because it lives in a "standard" submission directory. For BAPCtools, I've had to re-arrange many of these submissions to keep the tests passing (see https://github.com/RagnarGrootKoerkamp/BAPCtools/commit/8b12d05fa94c904d7ce008cbc4a47762132c8475) :stuck_out_tongue: